R/rdeephaven/R/client_wrapper.R

```diff
@@ -40,11 +42,12 @@ setMethod(
     if (auth_type == "anonymous") {
       options$set_default_authentication()
     } else if (auth_type == "basic") {
-      if (auth_token != "") {
-        verify_string("auth_token", auth_token, TRUE)
-        options$set_basic_authentication(auth_token)
+      if ((username != "") && (password != "")) {
+        verify_string("username", username, TRUE)
+        verify_string("password", password, TRUE)
+        options$set_basic_authentication(username, password)
```

Comment:
This is not doing what my review comment asked for.  We should be supporting both passwords AND auth_token for basic auth.  This is just doing one.  auth_token should be exactly like the code that got deleted here.